homeassistant/components/touchline/__init__.py

```diff
@@ -1 +1,60 @@
-"""The touchline component."""
+"""Platform for Roth Touchline floor heating controller."""
+from __future__ import annotations
+
+import logging
+import re
+
+from pytouchline import PyTouchline
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Instead of defining a new logger in each file, define it once in `const.py`:

```python
import logging

LOGGER = logging.getLogger(__package__)
```